### ISSUE TITLE
npm WARN deprecated babel-preset-babili@0.0.12: babili has been renamed to babel-minify. Please update to babel-preset-minify

### ISSUE THREAD

**Issue by jy95** [PhillMcKraken/enhanced-electron-react-boilerplate#7]
Hello,

Thanks for you work.

I tried today to test your projet, I got this warning

```sh
npm WARN deprecated babel-preset-babili@0.0.12: babili has been renamed to babel-minify. Please update to babel-preset-minify
```

I have done a npm ls to find out which dependancies use it : 

```
`-- babili-webpack-plugin@0.0.11
  `-- babel-preset-babili@0.0.12
```

PS: If you want to check out another dev dependancies , I use  https://david-dm.org/pbarbiero/enhanced-electron-react-boilerplate?type=dev

**Comment by pbarbiero** [comment]
Yes I need to update this, theres a lot of outdated deps and I want to move these projects to webpack 3.

Still trying to decide if I should switch over to uglifyjs-es or stick with babel-minify. Unfortunately `butternut` is no where near stable yet otherwise I think that would be my choice.

**Comment by pbarbiero** [comment]
It looks like uglify-es is the correct direction to go, especially with maturing webpack integration and plugins.

I am planning an update once webpack 4 and it's plugins mature.

**Comment by nurpax** [comment]
I did a quick & dirty webpack4 + other deps upgrade in https://github.com/pbarbiero/enhanced-electron-react-boilerplate/pull/15.  This doesn't touch the minifier but it's probably easy to change it now.

**Comment by nurpax** [comment]
Latest version of my PR switches to uglify-es (this was easy, the latest webpack-uglifyjs-plugin defaults to uglify-es (AFAICT).

I did a quick bundle size comparison:

With old code (babili):

```
-rw-r--r--   1 janne  staff   33462 Jul 16 00:58 bundle.css
-rw-r--r--   1 janne  staff  204633 Jul 16 00:58 bundle.js
drwxr-xr-x   5 janne  staff     160 Jul 15 01:53 font
-rw-r--r--   1 janne  staff     227 Jul 16 00:58 index.html
```

with webpack-uglifyjs-plugin (uglify-es, my PR):

```
-rw-r--r--   1 janne  staff   33462 Jul 16 00:59 bundle.css
-rw-r--r--   1 janne  staff  205617 Jul 16 00:59 bundle.js
drwxr-xr-x   5 janne  staff     160 Jul 15 01:53 font
-rw-r--r--   1 janne  staff     227 Jul 16 00:59 index.html
```

The new minifier produces 1kb larger bundle.  I didn't try tweaking any options for the minifier as this looked good enough for me.
